packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -123,17 +123,27 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     protected createInputTypes(model: ModelDefinition) {
         const modelFields = Object.values(model.graphqlType.getFields());
         const inputName = getInputTypeName(model.graphqlType.name);
-        
-        //TODO relationships?
+
         return new GraphQLInputObjectType({
             name: inputName,
-            fields: () => (modelFields.filter((field: GraphQLField<any, any>) => {
-                const fieldBaseType = getBaseType(field.type);
-
-                return !isObjectType(fieldBaseType);
-            }).reduce((fieldObj: any, current: any) => {
+            fields: () => (modelFields.reduce((fieldObj: any, current: any) => {
                 const fieldType = current.type;
-                fieldObj[current.name] = { type: getNullableType(fieldType), description: '' };
+
+                const relationshipField = findRelationship(current.name, model.relationships);
```

Comment:
Let's refactor this into core so we can simply answer the mapping question - what is the field name of the relationship same as we do for ID etc. Not sure why we need so much extra code here